jmix-core/core/src/main/java/io/jmix/core/entity/KeyValueEntity.java

```diff
@@ -97,7 +97,7 @@ public void setAttributeValue(@NonNull String name, Object value, boolean checkE
     }
 
     public KeyValueEntity() {
-        uuid = UuidProvider.createUuid();
+        uuid = UuidProvider.createUuidV7();
```

Comment:
Not customizable, but shouldn't be a problem because KeyValueEntity is in-memory only.